bigcases.py

```diff
@@ -67,7 +67,7 @@ def twitter_upload(self, image_list):
                 uploadable = BytesIO(res.content)
                 response = self.tw.upload_media(media=uploadable)
                 media_ids.append(response['media_id'])
-            except:
+            except Exception:
```

Comment:
If we believe this code is wrong but are not prepared to fix it right now, it should have an inline comment, at least `# xxx` and maybe something more verbose.